### PR TITLE
Add currency to the cart

### PR DESCRIPTION
**Pull request by Rexben001**
## Description
Add currency to the cart
## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update
## Checklist
- [x] I have performed a self-review of my own code
- [x] My code follows the style guidelines of this project
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes
## Link to Trello card
https://trello.com/c/4ncPeCvD/148-add-the-currency-to-the-cart-schema

### CODE REVIEW

**Comment by Rexben001** [comment]
[Add the currency to the cart schema](https://trello.com/c/4ncPeCvD/148-add-the-currency-to-the-cart-schema)

File: tests/routes/cart/getCart.test.js [hunk 1]
```diff
@@ -75,7 +75,14 @@ beforeAll(async () => {
...
       .post(`/api/store/${storeId}/cart`)
-      .send({ contents: [product1Id, product2Id], agreedPrice: 40, total: 400 })
+      .send({
+        contents: [
+          { product: product1Id, quantity: 3 },
+          { product: product2Id, quantity: 30 }
+        ],
+        agreedPrice: 40,
+        total: 400
+      })
```
Comment (aapetsi) [inline tests/routes/cart/getCart.test.js:86]: these tests need to be updated, since the currency and quantity fields are required

**Review by aapetsi — COMMENTED** [review]
The tests associated with the cart model needs to be updated

File: tests/routes/cart/getCart.test.js (hunk 1 above)
Comment (Rexben001) [reply tests/routes/cart/getCart.test.js:86]: The currency is added dynamically from the backend @aapetsi

**Review by aapetsi — APPROVED** [review]
All good

**Pull request merged**